ci/check_generated_docstrings.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# Check the generated docstrings
+#
+# pybind11 ships with mkdoc.py which extracts python docstrings
+# from the C++ doxygen comments. This is beneficial for maintaining
+# consistency and reducing duplication.
+#
+# In order to enable libsonata to be built without enforcing clang
+# as a dependency (required by mkdoc.py), we include the generated
+# docstrings in source control.
+#
+# In order to ensure they are being kept up-to-date, the CI process
+# regenerates them and checks there are no diffs.
+
+set -x
+set -euo pipefail
+
+if [[ -z $LIBCLANG_PATH ]]; then
+    echo "Expect a \$LIBCLANG_PATH in the environment, it should have the path to the libclang.so"
+    exit -1
+fi
+
+#VERSION=14c7d7002c71aba0623bee07f3258c2453f06262
+#PACKAGE=git+git://github.com/pybind/pybind11_mkdoc.git@$VERSION
+
+VERSION=0938515e0d4f9a5c2c5949dc46462175f54d6ea4
```

Comment:
@mgeplf I kept here your fork. Should I better revert to the original one?

Reply:
My fork is stale, I assume, best to point at the original one, I think.  They merged my change.

Reply:
Thank you, I changed to `master`